tools/cfn2ts/lib/codegen.ts

```diff
@@ -231,14 +231,12 @@ export default class CodeGenerator {
     this.code.line(' * Used in the @aws-cdk/cloudformation-include module.');
     this.code.line(' *');
     this.code.line(' * @experimental');
+    this.code.line(' * @internal');
     this.code.line(' */');
     // eslint-disable-next-line max-len
-    this.code.openBlock(`public static fromCloudFormation(scope: ${CONSTRUCT_CLASS}, id: string, resourceAttributes: any, options: ${CORE}.FromCloudFormationOptions): ` +
+    this.code.openBlock(`public static _fromCloudFormation(scope: ${CONSTRUCT_CLASS}, id: string, resourceAttributes: any, cfnParser: cfn_parse.CfnParser): ` +
```

Comment:
This is still supposed to take `options`. It's just `options`, instead of a `finder`, need to take a `CfnParser`.